src/components/tools/details/Drawer.js

```diff
@@ -38,10 +38,10 @@ const useStyles = makeStyles((theme) => ({
         [theme.breakpoints.up("lg")]: {
```

Comment:
I think you are right about `theme.breakpoints.up`, but the way that `theme.breakpoints.down` worked in v4 was confusing, so it's good that they changed that for v5 (not sure how I missed those breaking behavior docs for breakpoints before, or maybe I just forgot, but thanks for the link).

I think whoever added these `maxWidth` breakpoint rules was also confused by the v4 behavior, because in prod, this `maxWidth` does not go down to 25% until the screen width hits `xl`, because of the `theme.breakpoints.down` rule just below, yet this rule says it should go down to 25% for sizes `lg` and up.

I'm pretty sure that is a bug in prod, and the intention was for %25 maxWidth in `lg` and up, but 50% for widths between `sm` and `lg`. So the code-mod is preserving that bug in this PR, but I think we can fix it by reverting these changes for these rules, the others like it that I commented on.